garden_ai/garden.py

```diff
@@ -79,3 +81,65 @@ def _authenticate(self):
             on_refresh=self.auth_key_store.on_refresh,
         )
         return authorizer
+
+    def create_garden(self, **kwargs):
```

Comment:
Also wonder if this should just be a Garden constructor - not sure if having this as a method on GardenClient adds anything

Reply:
I was also thinking this was a little funny, but Owen's example made the logical user flow really nice? I am torn.

Reply:
I agree with both points, I think @BenGalewsky's vision earlier about `create_garden(title: str, authors: list[str], **kwargs)` is a nice compromise